src/ixdat/measurements.py

```diff
@@ -589,9 +610,25 @@ def __getitem__(self, key):
         series = self.get_series(key)
         # Finally, wherever we found the series, cache it and return it.
         # step 3.
-        self._cached_series[key] = series
+        self._cache_series(key, series)
         return series
 
+    def _cache_series(self, key, series):
+        """Cache `series` such that it can be looked up with its name or with `key`."""
+        self._cached_series[key] = series  # now it can be looked up with by `key`
+        # If the name of the series is not `key`, we can get in a situation where
```

Comment:
I'm a little in doubt about this change. Is this change changing what aliases may or may not be used for? I was under the assumption that aliases were either defined by readers or calibrations and were hence forth static, but this certainly changes that. Is that on purpose? And/or am I understanding the concept of alias wrong? Additionally, how would this compare to simply adding the series to `_cached_series` under their name also? Or will that somehow mess up because because of a possibility for more than one with the same name?

Reply:
Good point. I hadn't thought of `aliases` as static, but I hadn't thought of it as  necessarily dynamic either. No reason to make it dynamic when there's another option. simple better than complex better than complicated, and `cached_series` is clearly already dynamic.
I've changed it to your suggestion.